tests/base.py

```diff
@@ -101,6 +101,7 @@ def setUp(self):
         self.app.preprocess_request()
         g.db = create_session(
             self.app.config['DATABASE_URI'], drop_tables=True)
+        g.db.execute('pragma foreign_keys=on')
```

Comment:
@T1duS can you put a comment here for this code. so that we can understand its reason later.